### PR TITLE
TPROD 331 - GrapesJsBuilder: migrate views from PHP Engine to Twig

### PR DESCRIPTION
**Pull request by LordRembo**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
- Replaced the html.php templates with html.twig version
- Replaced references to the php templates with the new twig versions
- Did NOT remove references to slideshow.html.php `(MauticPageBundle:Page:Slots/slideshow.html.php)`. I believe this is okay, as another ticket which has been recently finished and merged (eg. PROD-322 PageBundle) also still references that php template. We can remove/replace these references when the component responsible is tackled (possibly CoreBundle? not sure)

#### Steps to test this PR:
In short: check that the GrapesJS builder still works and produces the expected content and html

- Open up a Mautic instance in the browser
- Make sure you have at least 1 contact in Mautic Contact
- In Channels -> Emails add a new Segment Email
- Select a theme made with some nice html or mjml (eg. Paprika)
- Open the Builder to see that everything in your template renders nicely
- Make some additions (eg. add a 2 col with a Text block and an image block)
- Save
- Open it again to see if it all still renders
- Close & send an email to one of your contacts
- Check that the mail is delivered as an html mail
 

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.

### CODE REVIEW

**Comment by LordRembo** [comment]
@cla-bot check

**Comment by mabumusa1** [comment]
Hi @TechBill, would be great if you can test this PR

**Review by mabumusa1 — CHANGES_REQUESTED** [review]
The test did not work for me, I created a new email and selected different themes, but I always end up with the same results. Attached screenshot

image

When I close the builder I get 

image

**Comment by LordRembo** [comment]
Oh wow, alright. I'll check why my test environment didn't show this and fix it.

**Comment by LordRembo** [comment]
I might not get to it before Monday. Will see.

**Review by mabumusa1 — APPROVED** [review]
I tested this PR and it worked as expected, emails are rendered as HTML as usual
image

**Comment by TechBill** [comment]
Tested this PR 

The email and template creation worked as intended. 

However, unable to test the email part since Mailhog was not loading for me on Gitpod.

**Comment by mabumusa1** [comment]
@LordRembo I think this PR is almost ready, can you increase the test coverage so we can merge it?

**Comment by sdoering** [comment]
Renders nicely in the Builder interface:

image

But I receive an error, that the mailhog port 8025 can't be reached when testing on gitpod.io. So I can't see the effect of the changes currently. 

EDIT: 
A new instance of the PR in gitpod (new workspace) did not solve the mailhog issue. So can't test the resulting email.

**Comment by sdoering** [comment]
Tested it again. This time with Mailhog working. 

Worked flawlessly and Mail looks great in Mailhog. 💚
